StripePaymentSheet/StripePaymentSheetTests/PaymentSheet/PaymentSheetFormFactorySnapshotTest.swift

```diff
@@ -395,38 +390,14 @@ final class PaymentSheetFormFactorySnapshotTest: STPSnapshotTestCase {
 }
 
 extension PaymentSheetFormFactorySnapshotTest {
-    private func usAddressSpecProvider() -> AddressSpecProvider {
-        let specProvider = AddressSpecProvider()
-        specProvider.addressSpecs = [
-            "US": AddressSpec(
-                format: "NOACSZ",
-                require: "ACSZ",
-                cityNameType: .city,
-                stateNameType: .state,
-                zip: "",
-                zipNameType: .zip
```

Comment:
Is this what is causing all the snapshot tests to default "State" to Alabama?

Reply:
Yes. The current state is better; it's ~not possible for PaymentSheet to look like the old snapshots where US state is a textfield and not a dropdown.